core/src/config/sequelize.ts

```diff
@@ -18,6 +18,7 @@ require("pg").defaults.parseInt8 = true;
 
 export const DEFAULT = {
   sequelize: (config) => {
+    const env = process.env.NODE_ENV;
```

Comment:
On most developer machines, NODE_ENV will be `undefined`, so maybe

```suggestion
    const env = process.env.NODE_ENV ?? "development";
```